app/src/app/components/contents/common/DataTable.tsx

```diff
@@ -16,24 +17,40 @@ interface DataTableProps<TData> {
   pagination?: {
     pageSize: number;
   };
+  visibility?: {
+    columnVisibility: Record<string, boolean>;
+    setColumnVisibility: Dispatch<SetStateAction<Record<string, boolean>>>;
```

Comment:
Why does the table need setColumnVisibility? We're not setting from with the table (yet..?)